src/styles/Theme.tsx

```diff
@@ -30,12 +30,19 @@ const theme: DefaultTheme = {
         height: '2',
       },
       sectionTitle: {
-        min: '24px',
-        size: '36px',
-        max: '2.4rem',
+        min: '20px',
+        size: '24px',
+        max: '2rem',
         weight: '600',
         height: '',
```

Comment:
what does the height  ' ' do ah

Reply:
I think it's part of the attributes that must be created when I made this, and since I have no use for that attribute now I initialised to `''` as with the other `fontTypes` :smile: